requirements/main.in

```diff
@@ -1,6 +1,7 @@
 alembic>=0.7.0
 Automat
 argon2-cffi
+b2sdk
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought:** ‏sad that their SDK includes a whole dep [for one call](https://github.com/Backblaze/b2-sdk-python/blob/56c82aa14a918bcd89470e32c84744d737edf9b8/b2sdk/b2http.py#L117-L119)

Reply:
https://github.com/Backblaze/b2-sdk-python/pull/386